### ISSUE TITLE
Problem connecting to My3 website

### ISSUE THREAD

**Issue by dalb8** [DamienOReilly/PrepayCredit#8]
I was on O2 and my credentials for that website now work on login.three.ie. However v0.15 reports an error: damp.three.ie.util.PrepayException: Unexpected response from server. Logcat says nothing.

**Comment by DamienOReilly** [comment]
Hi,

login.three.ie is new to me. It must be for o2 customers. I can only partially login with it, and its kinda o2 branded.

Existing 3 users use https://my3account.three.ie and hence my app uses that. Can you try login via that URL and report what you observe?

Thanks.

**Comment by dalb8** [comment]
Yeah sorry I forgot, I already tried that URL and it didn't work. It was pure skill I found the login.three.ie :-)

**Comment by DamienOReilly** [comment]
What does it say when you try logging in via that URL?

I have a feeling the logging in sequence goes a bit different for o2 users now.

**Comment by dalb8** [comment]
something like username or password incorrect.

Previously, I thought I would need a new password so they sent me a text with password that wasn't even 7 digits long and had no numbers. Of course that didn't work. God, what a shower...

**Comment by DamienOReilly** [comment]
Hmm I will have to think about how to overcome this issue!

**Comment by dalb8** [comment]
Actually it was 7 chars what they sent me but no numbers. So, the red button 'sign up' points https://my3account.three.ie/Sign_up (defunct, in my case) and the blue 'register' button points https://login.three.ie/register. Only in the FAQ at the latter does it say the o2 credentials work.

**Comment by dalb8** [comment]
sorry, s/blue/grey in last comment

**Comment by dalb8** [comment]
I suppose the first one's only for dongles anyway.

**Comment by DamienOReilly** [comment]
Yeah the whole thing is a mess. I can actually log in via login.three.ie
with my credentials. Not sure what happened the first time. I will see what
I can do with it.

**Comment by DamienOReilly** [comment]
My3account.three.ie was always login URL for everyone, broadband only,
prepay and bill pay. Before that it was my3.three.ie

login.three.ie seems to be introduced as part of the o2 merger.

**Comment by DamienOReilly** [comment]
Here is how the usage page looks for me: 
![](http://i.imgur.com/9mUZ3sx.jpg)

I looked at an o2 account and the page looks very different, so there will be a bit of work in this.

**Comment by DamienOReilly** [comment]
O2 on 3:

![](http://i.imgur.com/QxabjLt.jpg)

**Comment by DamienOReilly** [comment]
Just looking back at this. Do you have a screenshot of what it looks like when you log in.

I have no visibility what the different plans for o2 look like on 3's page.

**Comment by dalb8** [comment]
Typically the page goes blank once it loads fully. I snapped what is shown as it's loading (minus my number)/
![git](https://cloud.githubusercontent.com/assets/726844/7914084/86d0cb54-086c-11e5-8bb7-239ac280adde.png)

**Comment by DamienOReilly** [comment]
Ok so the page is actually broken for you on My3 ?

**Comment by dalb8** [comment]
I just tried again on Android Firefox and it's working. Similar to screenshot above. Also a bit about my 30 day payg data allowance. Can upload screenshot if necessary when I get to a desktop

**Comment by dalb8** [comment]
![screenshot-20150604-171439](https://cloud.githubusercontent.com/assets/726844/7990563/b882e176-0ae9-11e5-8465-f9fbe015d986.png)
![screenshot-20150604-171452](https://cloud.githubusercontent.com/assets/726844/7990562/b881fbd0-0ae9-11e5-8222-53a2fe2f6b67.png)
